<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>100</y>
      <width>301</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="OUT" stretch="0,1">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ANALÓGICO</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DIGITAL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Out_values" stretch="1,1">
       <item>
        <layout class="QVBoxLayout" name="Out_Avalues">
         <item>
          <widget class="QLCDNumber" name="Out_A1_value">
           <property name="digitCount">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="Out_A1_slider">
           <property name="toolTipDuration">
            <number>0</number>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="Out_A2_value">
           <property name="digitCount">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="Out_A2_slider">
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="Out_Dvalues">
         <item>
          <widget class="QLCDNumber" name="Out_D1">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="Out_D2">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="Out_D3">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="Out_D4">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="Out_D5">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="Out_D6">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="Out_D7">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="Out_D8">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>321</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="IN" stretch="0,1">
     <item>
      <layout class="QHBoxLayout" name="In_names" stretch="0,0">
       <item>
        <widget class="QLabel" name="In_analogText">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>ANALÓGICO</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="In_digText">
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>DIGITAL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="In_values" stretch="0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="In_Avalues">
         <item>
          <widget class="QLCDNumber" name="In_A1">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="digitCount">
            <number>4</number>
           </property>
           <property name="value" stdset="0">
            <double>0.000000000000000</double>
           </property>
           <property name="intValue" stdset="0">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_A2">
           <property name="digitCount">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_A3">
           <property name="digitCount">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_A4">
           <property name="digitCount">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_A5">
           <property name="digitCount">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_A6">
           <property name="digitCount">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_A7">
           <property name="digitCount">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_A8">
           <property name="digitCount">
            <number>4</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="In_Dvalues">
         <item>
          <widget class="QLCDNumber" name="In_D1">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_D2">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_D3">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_D4">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_D5">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_D6">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_D7">
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="In_D8">
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="digitCount">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>1131</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="labelSalidas">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>SALIDAS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelEntradas">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>20</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>ENTRADAS</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>130</y>
      <width>161</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="outB1">
       <property name="text">
        <string>Out 1</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="outB2">
       <property name="text">
        <string>OUT 2</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="outB3">
       <property name="text">
        <string>OUT 3</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="outB4">
       <property name="text">
        <string>OUT 4</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="outB5">
       <property name="text">
        <string>OUT 5</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="outB6">
       <property name="text">
        <string>OUT 6</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="outB7">
       <property name="text">
        <string>OUT 7</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="outB8">
       <property name="text">
        <string>OUT 8</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
